test/mock_sdk.dart

```diff
@@ -377,7 +377,7 @@ class  ScriptElement {
       String libraryPath = provider.convertPath(library.path);
       if (filePath == libraryPath) {
         try {
-          resource.File file = provider.getResource(filePath);
+          resource.File file = provider.getResource(filePath) as resource.File;
```

Comment:
Consider using `getFile` rather than `getResource`, both here and below.

Reply:
Thanks!